tests/test_symbolic/test_tensor.py

```diff
@@ -281,3 +281,41 @@ def test_lambdify_with_args():
     assert_allclose(arr, cmp_arr, rtol=0.0, atol=1e-7)
     for e1, e2 in zip(args, (y, x)):
         assert simplify(e1 - e2) == 0
+
+
+def test_derivative():
+    x, y = symbols("x y")
+    test_list = [[x, y], [y, x]]
+    obj1 = BaseRelativityTensor(test_list, syms=[x, y], config="ll")
+
+    test_list = [[1, 0], [0, 1]]
+    obj2 = BaseRelativityTensor(test_list, syms=[x, y], config="ll")
+    # call using symbol
+    derived = obj1.derivative(x)
+    assert derived.arr == obj2.arr
+    # call using list
+    derived = obj1.derivative(x)
+    assert derived.arr == obj2.arr
+    # call using BaseRelativityTensor
+    test_list = [1, 0]
```

Comment:
I think you try to test something else, but you forgot it and repeted the code:

`    # call using list
    derived = obj1.derivative(x)
    assert derived.arr == obj2.arr`

Reply:
yep, totally. I'll fix this in the next commit. Thanks!